README.md

```diff
@@ -112,6 +112,16 @@ Ott Emacs directory.
 (require 'ott-mode)
 ```
 
+For installations using OPAM on \*nix systems, it is sufficient to use the following code, which will call `opam config var prefix` at load-time.
+
+```ELisp
+(add-to-list 'load-path
+              (concat (let ((coding-system-for-read 'utf-8))
+                        (shell-command-to-string "printf %s \"$(opam config var prefix)\""))
+                      "/share/emacs/site-lisp"))
```

Comment:
I use a slightly shorter emacs lisp snippet (which uses `opam config var share` directly, no `printf %s`, instead`substring`)
```
(setq opam-share (substring (shell-command-to-string "opam config var share") 0 -1))
(add-to-list 'load-path (concat opam-share "/emacs/site-lisp"))
```

Reply:
I've stolen `(coding-system-for-read 'utf-8)` from some other code (not sure if ever needed), and `printf` is my elisp being bad. I'll switch to yours.